test/registry/simple_registry_test.js

```diff
@@ -211,6 +212,23 @@ contract("SimpleRegistry", accounts => {
     assert(!isFailed, "Should have thrown an exception");
   });
 
+  it("should not allow to transfer to address 0x0", async () => {
+    let isFailed = false;
+
+    try {
+      await simpleReg.transfer(name, DEFAULT_ADDRESS, {
+        from: accounts[1]
+      });
+      isFaield = true;
```

Comment:
fix typo

Reply:
I copied that part of code, I corrected it everywhere